scripts/download-rc.js

```diff
@@ -1,136 +1,161 @@
 'use strict';
-const fs = require('fs');
-const os = require('os');
-const {spawnSync} = require('child_process');
-const rcParams = require('./rc-params');
+const HZ_VERSION = '4.1.1';
+const HZ_TEST_VERSION = '4.1.1';
+const HAZELCAST_TEST_VERSION = HZ_TEST_VERSION;
+const HAZELCAST_VERSION = HZ_VERSION;
+const HAZELCAST_ENTERPRISE_VERSION = HZ_VERSION;
+const HAZELCAST_ENTERPRISE_TEST_VERSION = HZ_VERSION;
+const HAZELCAST_RC_VERSION = '0.8-SNAPSHOT';
+const SNAPSHOT_REPO = 'https://oss.sonatype.org/content/repositories/snapshots';
+const RELEASE_REPO = 'http://repo1.maven.apache.org/maven2';
+const ENTERPRISE_RELEASE_REPO = 'https://repository.hazelcast.com/release/';
+const ENTERPRISE_SNAPSHOT_REPO = 'https://repository.hazelcast.com/snapshot/';
 
-const ON_WINDOWS = os.platform() === 'win32';
-const HAZELCAST_ENTERPRISE_KEY = process.env.HAZELCAST_ENTERPRISE_KEY ? process.env.HAZELCAST_ENTERPRISE_KEY : '';
+const downloadRC = () => {
```

Comment:
this is the main function of this module. it is exported and called on direct invoke of this module